apps/docs/src/guide/errors/index.md

```diff
@@ -1,3 +1,270 @@
 # Errors
 
-All errors thrown from our SDK are instances of the `FuelError` class, which will have an accompanying `ErrorCode`. You can find a table with all our expected error codes and their descriptions [here](./error-codes.md).
+All errors thrown from our SDK are instances of the `FuelError` class, which will have an accompanying `ErrorCode`.
+
+## Error Codes
+
+Here is a list of the expected error codes the SDK can throw. These error codes are used to help understand the error that has been thrown with potential resolutions.
+
+### `ABI_MAIN_METHOD_MISSING`
+
+When your ABI does not have a main method.
+
+This can be resolved by adding a main method to your ABI.
```

Comment:
```suggestion
This can be resolved by adding a `main` method to your ABI. This is prevalent in scripts and predicates that must contain a `main` method.
```